examples/svelte/src/types.d.ts

```diff
@@ -0,0 +1,29 @@
+declare module "https://link-sdk.codat.io" {
+  export interface ConnectionCallbackArgs {
+    connectionId: string;
+  }
+
+  export interface ErrorCallbackArgs {
+    correlationId?: string;
+    message?: string;
+    errorCode?: number;
+    userRecoverable: boolean;
+  }
+
+  export type CodatLinkProps = {
+    companyId: string;
+    onClose: () => void;
+    onError: (args: ErrorCallbackArgs) => void;
+    onFinish: () => void;
+    onConnection: (args: ConnectionCallbackArgs) => void;
+  };
+
+  //   The error message you encountered, "A 'declare' modifier cannot be used in an already ambient context," typically occurs when you are attempting to use the declare keyword inside a module declaration (declare module ...) but the code is already in an ambient context (e.g., a module or a script that is already considered ambient).
+
+  // In the example you provided, it seems like you are trying to declare a module for the "https://link-sdk.codat.io" library. However, since the module declaration itself is already considered ambient (outside the scope of the TypeScript module system), you don't need the declare keyword inside it.
```

Comment:
What are these comments and do we need them?

Reply:
Thanks for spotting this, got carried away with the refactor to actions yesterday and completely forgot about it!

https://github.com/codatio/sdk-link/blob/main/snippets/types.d.ts line 21 has 'declare' after export, but I was having the following error due to it. Wanted to clarify with you if 'declare' keyword should be deleted in the example file.